src/containerapp/azext_containerapp/_sdk_models.py

```diff
@@ -5019,18 +5066,58 @@ class ContainerAppsJob(TrackedResource):
-        'template': {'key': 'properties.jobTemplate', 'type': 'JobTemplate'},
+        "environment_id": {"key": "properties.environmentId", "type": "str"},
+        "workload_profile_name": {"key": "properties.workloadProfileName", "type": "str"},
+        'configuration': {'key': 'properties.configuration', 'type': 'JobConfiguration'},
+        'template': {'key': 'properties.template', 'type': 'JobTemplate'},
+        "event_stream_endpoint": {"key": "properties.eventStreamEndpoint", "type": "str"},
     }
 
-    def __init__(self, **kwargs):
-        super(ContainerAppsJob, self).__init__(**kwargs)
-        self.identity = kwargs.get('identity', None)
+    def __init__(
+        self,
+        *,
+        location: str,
+        tags: Optional[Dict[str, str]] = None,
+        extended_location: Optional["_models.ExtendedLocation"] = None,
+        identity: Optional["_models.ManagedServiceIdentity"] = None,
+        managed_by: Optional[str] = None,
+        managed_environment_id: Optional[str] = None,
+        environment_id: Optional[str] = None,
+        workload_profile_name: Optional[str] = None,
+        configuration: Optional["_models.JobConfiguration"] = None,
+        template: Optional["_models.JobTemplate"] = None,
+        **kwargs: Any
+    ) -> None:
+        """
+        :keyword tags: Resource tags.
+        :paramtype tags: dict[str, str]
+        :keyword location: The geo-location where the resource lives. Required.
+        :paramtype location: str
+        :keyword identity: managed identities for the Container Apps Job to interact with other Azure
+         services without maintaining any secrets or credentials in code.
+        :paramtype identity: ~commondefinitions.models.ManagedServiceIdentity
+        :keyword managed_environment_id: Deprecated. Resource ID of the Container Apps Job's environment.
+        :paramtype managed_environment_id: str
+        :keyword environment_id: Resource ID of environment.
+        :paramtype environment_id: str
+        :keyword workload_profile_name: Workload profile name to pin for container app execution.
+        :paramtype workload_profile_name: str
+        :param jobConfiguration: Non versioned Container Apps job configuration
+            properties.
+        :type jobConfiguration: ~commondefinitions.models.JobConfiguration
+        :param jobTemplate: Container App versioned application definition.
+        :type template: ~commondefinitions.models.JobTemplate
+        """
+        super().__init__(tags=tags, location=location, **kwargs)
+        self.identity = identity
+        self.managed_by = managed_by
         self.provisioning_state = None
-        self.managed_environment_id = kwargs.get('managed_environment_id', None)
-        self.configuration = kwargs.get('configuration', None)
-        self.template = kwargs.get('template', None)
-        self.outbound_ip_addresses = None
```

Comment:
Do you sure to remove the outbound_ip_addresses?

Reply:
I rechecked, added it again.